src/components/ResponsiveAppBar.jsx

```diff
@@ -102,6 +113,7 @@ const ResponsiveAppBar = () => {
                     onClick={handleCloseNavMenu}
                     component={Link}
                     to={`/${page.url}`}
+                    data-welcome-tour={page.url}
```

Comment:
Those `data-welcome-tour` are here to highlight elements. Since those are hidden by the menu, we do not highlight them

```suggestion
```